src/components/utils/pin_validator.ts

```diff
@@ -0,0 +1,33 @@
+const isOibValid = (input: number) => {
```

Comment:
We could rename the function to `pinValidator`, or we could rename the method and file to `oibValidator` and `oibValidator.ts`, respectively, since the method is specifically tied to OIB validation.

Reply:
Small tip regarding return types of functions, TypeScript is smart enough to recognize the return type of a function, which in this case of this method would be a boolean. However, it is usually preferable for the developer to explicitly define the return type of the method, which in this case would be `(input: number): boolean => {`.